build.sbt

```diff
@@ -749,23 +740,15 @@ lazy val junit = project.in(file("test") / "junit")
 
 
 lazy val scalacheck = project.in(file("test") / "scalacheck")
-  .dependsOn(library, reflect, compiler, scaladoc, scalacheckLib)
+  .dependsOn(library, reflect, compiler, scaladoc)
   .settings(commonSettings)
   .settings(disableDocs)
   .settings(skip in publish := true)
   .settings(
-    // Enable forking to workaround https://github.com/sbt/sbt/issues/4009. We also need a lazy Framework
-    // written in Java to further reduce ClassLoader problems when bootstrapping:
+    // Enable forking to workaround https://github.com/sbt/sbt/issues/4009.
     fork in Test := true,
-    testFrameworks := Seq(TestFramework("org.scalacheck.LazyFramework")),
-    // When we switch to sbt 1.3.0 it should be possible to set the ClassLoaderLayeringStrategy instead:
-    //classLoaderLayeringStrategy in Test := ClassLoaderLayeringStrategy.Flat,
-    // customise framework for early access to https://github.com/rickynils/scalacheck/pull/388
-    // TODO remove this when we upgrade scalacheck
-    //testFrameworks := Seq(TestFramework("org.scalacheck.CustomScalaCheckFramework")),
     javaOptions in Test += "-Xss1M",
-    //Make scalacheck print full stack traces
-    testOptions in Test += Tests.Argument(TestFramework("org.scalacheck.LazyFramework"), "-verbosity", "2"),
```

Comment:
why removed? (the full-stack trace part, I mean)

Reply:
I don't think it's necessary, but I can put it back.